ansible_collections/arista/avd/examples/campus-fabric/README.md

```diff
@@ -0,0 +1,390 @@
+
+In Campus Networks it is uncommon to have a dedicated out-of-band management network in each IDF. You can easily disable configuring Management0 interface and vrf MGMT, by adding these variables to the DC1_LEAFS.yml group_vars.
+
+``` yaml
+mgmt_gateway: null
+mgmt_interface: null
+mgmt_interface_vrf: default
+```
+
+???+ note
+
+    If you switch to using in-band management only, update other variables that reference vrf MGMT to use the vrf default.
+
+## Build Fabric Topology
+
+AVD Fabric Input Variables
+
+To apply AVD input variables to the nodes in the fabric, we make use of Ansible group_vars. How and where you define the variables is your choice. The group_vars table below is one example of AVD fabric variables.
+
+| group_vars/              | Description                                   |
+| ------------------------ | --------------------------------------------- |
+| DC1.yml                  | Global settings for all devices               |
+| DC1_FABRIC.yml           | Fabric, Topology, and Device settings         |
+| DC1_SPINES.yml           | Device type for Spines                        |
+| DC1_LEAFS.yml            | Device type for Leafs                         |
+| DC1_NETWORK_SERVICES.yml | VLANs/SVIs                                    |
+| DC1_NETWORK_PORTS.yml    | Port Profiles and Network Port Ranges         |
+
+The tabs below show the Ansible **group_vars** used in this example.
+
+=== "DC1"
+    At the top level (DC1), the following variables are defined in **group_vars/DC1.yml**. These Ansible variables apply to all nodes in the fabric and is a common place to set AAA, users, NTP, and management interface settings. Update local_users and passwords for your environment.
+
+    You can create a sha512_password by creating a username and password on a switch and copy/paste it within double quotes here.
+
+    ``` yaml
+    --8<--
+    examples/campus-fabric/group_vars/DC1.yml
+    --8<--
+    ```
+
+=== "DC1_FABRIC"
+    At the Fabric level (DC1_FABRIC), the following variables are defined in **group_vars/DC1_FABRIC.yml**. The fabric name, design type (l2ls), spine and leaf defaults, ansible authentication, and interface links are defined at this level. Other variables you must supply include: spanning-tree mode and priority along with an mlag IP pool. Typically, and IDF has a unique set of VLANs. You may use the `filter.tags` variable to constrain which vlans are built on an IDF node.
+
+    Variables applied under the node key type (spine/leaf) defaults section are inherited to nodes under each type. These variables may be overwritten under the node itself.
+
+    The spine interface used by a particular leaf is defined from the leaf's perspective with a variable called `uplink_switch_interfaces`. For example, LEAF2A has a unique variable `uplink_switch_interfaces: [Ethernet49/1, Ethernet49/1]` defined. This means that LEAF2A is connected to SPINE1's Ethernet49/1 and SPINE2's Ethernet49/1, respectively.
```

Comment:
Is the spacing in the beginning on purpose? In the rendered docs this will not show up.

Reply:
Yes, it is part of the indentation needed for the Tabs to work properly.